.vscode/launch.json

```diff
@@ -44,6 +44,24 @@
                 "${workspaceFolder}/out/**/*.js"
             ]
         },
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "frameworkTest",
+            "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
+            "args": [
+                "--debug-brk", "--inspect",
```

Comment:
Should not be necessary in a launch config

Reply:
Agreed, and removed.